src/sentry/models/grouptombstone.py

```diff
@@ -39,6 +39,8 @@ class GroupTombstone(Model):
         blank=True, null=True
     )
     actor_id = BoundedPositiveIntegerField(null=True)
+    times_seen = BoundedPositiveIntegerField(db_default=0)
+    last_seen = models.DateTimeField(default=None, null=True)
```

Comment:
Nit: for `last_seen`, If you just want the default to be None, you don't need to include this default at all.

If you want it to just be set to the current date/time when the row is first created, you can use `auto_now_add=True` instead of using a default.

Reply:
I was planned to write another migration just to do that for existing rows and you just saved me from that! thank youuu